src/models/init/associations.ts

```diff
@@ -38,6 +39,7 @@ export function associate(models: Models) {
   Stall.hasMany(Product, { foreignKey: 'stallId', onDelete: 'cascade', hooks: true });
   Stall.hasMany(Review, { foreignKey: 'stallId', onDelete: 'cascade', hooks: true });
   Stall.hasMany(Favourite, { foreignKey: 'stallId', onDelete: 'cascade', hooks: true });
+  Stall.hasMany(CategoryStall, { foreignKey: 'stallId', onDelete: 'cascade', hooks: true });
```

Comment:
Is it possible to create APIs to deal with the categories of stalls? Actually, you just need one -> 
```
POST /stalls/1/setCategories
// send a list of categories ID to set
[1, 2, 3]
```
Removal and adding of categories to a specific store can be done through this API